autoconfigure/pom.xml

```diff
@@ -33,6 +35,7 @@
 
   <modules>
     <module>collector-eventhub</module>
+    <module>storage-applicationinsights</module>
```

Comment:
collector is the transport protocol(default Http) you listen on ... for receiving spans. Right now we are using Http - so, I guess we might not have a separate AI collector unless you have a better business case where AI collector helps. 
Collectors like Eventhub could have many clients other than storage.